python/isogram/isogram.py

```diff
@@ -0,0 +1,6 @@
+def is_isogram(string): 
+    lowercase_string = string.lower()
+    for alphabet in lowercase_string:
+        if lowercase_string.count(alphabet) > 1 and alphabet.isalpha():
```

Comment:
```
if  alphabet.isalpha() and lowercase_string.count(alphabet) > 1:
```
the count operation is only done when the character is an alphabet. I would also rename `alphabet` to `character`